sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -844,6 +851,13 @@ export interface BlobDownloadToBufferOptions extends CommonOptions {
    * @memberof BlobDownloadToBufferOptions
    */
   concurrency?: number;
+  /**
+   * Customer Provided Key Info.
+   *
+   * @type {CpkInfo}
+   * @memberof BlobDownloadToBufferOptions
+   */
+  customerProvidedKey?: CpkInfo;
```

Comment:
Consider to add test, if this is fix brought by this change.

Reply:
Irrelevant to this feature. We somehow missed it.

Reply:
Added a test case.